mcpi/util.py

```diff
@@ -1,4 +1,7 @@
-import collections
+try:
+    import collections.abc as collections
```

Comment:
What does the "abc" mean? What is that naming convention, I have seen it in other places b4.

Reply:
it means "abstract base class", more info [here](https://docs.python.org/3/glossary.html#term-abstract-base-class)